azure-pipelines.yml

```diff
@@ -35,7 +35,7 @@ stages:
     - template: .ci/build-platform.yml
       parameters:
         platform: Linux
-        vmImage: ubuntu-16.04
+        vmImage: ubuntu-18.04
```

Comment:
I'm personally not opposed to bumping this here, but if Esy is gonna keep distributing prebuilt binaries then it should probably make sure they're built on an older Linux (cc: @ManasJayanth)

Reply:
I'm not sure I understand the issue here - Dune 1.x doesn't build on older Ubuntu? I cant understand why.

Reply:
The issue seems to have been that Esy didn't run with a libc < 2.27

Reply:
Got it. This could be reported on esy's issue tracker. Thank you.